[Microsoft Graph User] Handle User Queries with Unsupported Characters (#15586)
demisto/content: Packs/MicrosoftGraphUser/ReleaseNotes/1_3_15.md

#### Integrations
##### Azure Active Directory Users

> - Fixed an issue where running the command **msgraph-user-get** on a user containing any of the unsupported characters **%&*+/=?`{|}** would result in an error.

Reviewer: ```suggestion
- Fixed the misleading error message when running the ***msgraph-user-get*** command on a user containing an unsupported character.
```
